src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodRecordsAndCompactCtors.java

```diff
@@ -5,7 +5,7 @@
 accessModifiers = (default)public, protected, package, private
 allowMissingParamTags = (default)false
 allowMissingReturnTag = (default)false
-tokens = METHOD_DEF, CTOR_DEF, ANNOTATION_FIELD_DEF, COMPACT_CTOR_DEF, RECORD_DEF, CLASS_DEF
+tokens = METHOD_DEF, CTOR_DEF, ANNOTATION_FIELD_DEF, COMPACT_CTOR_DEF
```

Comment:
this means breaking compatibility for users.
this check is very old, so users might have all this tokens.

no good reason to have such tokens, but it will still be compatibility issue - exception in run time after upgrade of checkstyle.

Reply:
@rnveach , are we in agreement ?

Reply:
I don't really see this as a breaking change, but you will do what you want anyways.